dscreator/datasets/timeseries/glomma.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from dataclasses import dataclass
+
+import xarray as xr
+from dataclasses import asdict
+
+from dscreator import utils
+from dscreator.cfarray.attributes import VariableAttrs, DatasetAttrsDiscrete
+from dscreator.datasets.base import TimeseriesDatasetBuilder
+
+
+@dataclass
+class BaterodBuilder(TimeseriesDatasetBuilder):
```

Comment:
I think it would be better to be consistent with naming here, either Glomma or Baterod. If we keep Baterod here, we should use "baterod" in the __init__.py